patternfly/patternfly-elements
# chore: Add nvmrc

Add an nvmrc file pointing to v12.16.3.

### Related issues

- [#1506](https://github.com/patternfly/patternfly-elements/issues/1506) Add nvmrc


### Testing instructions

1. cd to the patternfly-elements project
2. `nvm use`
3. Should change to v12.16.3

### Ready-for-merge Checklist



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
